rotkehlchen/db/filtering.py

```diff
@@ -281,6 +282,14 @@ def prepare(self) -> tuple[list[str], list[Any]]:
         return ['location=?'], [self.location.serialize_for_db()]
 
 
+@dataclass(init=True, repr=True, eq=True, order=False, unsafe_hash=False, frozen=False)
+class DBProductFilter(DBFilter):
```

Comment:
Do you need to create a new filter? Can't you use `DBEqualsFilter`? I don't think we should be creating new filters for everything but try to consolidate